frontend/javascripts/libs/utils.ts

```diff
@@ -746,9 +747,33 @@ export function convertDecToBase256(num: number): Vector4 {
 
   [tmp, a] = divMod(tmp); // eslint-disable-line prefer-const
 
-  // Big endian
-  return [a, b, g, r];
+  // Little endian
+  return [r, g, b, a];
```

Comment:
The order used to be reversed by the caller which is why I changed it here. It wasn't a bug.